include/jsonbuilder/JsonBuilder.h

```diff
@@ -1479,6 +1545,42 @@ Exchanges the contents of two JsonBuilder objects.
 */
 void swap(JsonBuilder&, JsonBuilder&) throw();
 
+/*
+UUID object, network byte order (big-endian), compatible with uuid_t.
+Does NOT use the same byte order as the Windows GUID type.
+*/
+struct UuidStruct
+{
+    char unsigned Data[16]; // Compatible with uuid_t from libuuid.
+};
+
+/*
+DateTime, number of 100ns intervals since 1601, compatible with Win32 FILETIME.
+*/
+struct TimeStruct
```

Comment:
Note: We need our time representation to be platform-independent. The meaning of `std::chrono::system_clock::time_point` is not platform-independent, so it can't be used as the underlying representation -- the frequency and the clock starting point vary depending on which C++ library you link against.

I decided to go with FILETIME as the underlying representation: 64-bit, little-endian, 100ns-units, 1601 starting point.